spec/fixtures/purl_fixtures.rb

```diff
@@ -963,6 +963,32 @@ def single_video_purl
     XML
   end
 
+  def video_purl_with_vtt
+    <<-XML
+      <publicObject>
+        <identityMetadata>
+          <objectLabel>Title of the single video</objectLabel>
+        </identityMetadata>
+        <contentMetadata type="media">
+          <resource sequence="1" id="abc123_1" type="video">
+            <file id="abc_123.mp4" mimetype="video/mp4" size="152000000"></file>
+            <file id="abc_123_cap.vtt" mimetype="text/plain" size="176218"></file>
```

Comment:
Isn't `text/vtt` the registered mime-type for vtt files?

Reply:
Yes it is - nice catch.